### PR TITLE
Added custom initial and custom close icon, with deprecated code migrations

### PR DESCRIPTION
**Pull request by csof3cen**
Hi! This pull request contains new features : you can now customize the initial and close icon. I also migrated the deprecated code. I'll add other features soon, more customization, more animations...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +27,8 @@ class MyApp extends StatelessWidget {
       home: Scaffold(
         body: Placeholder(),
         floatingActionButton: ExpendableFab(
+          icon: Icon(Icons.favorite),
+          // closeIcon: Icon(Icons.close),
```
Comment (ahmadalibaloch) [inline README.md]: can you remove this commented code plz

File: example/lib/main.dart [hunk 2]
```diff
@@ -32,6 +31,8 @@ class _MyAppState extends State<MyApp> {
         ),
         floatingActionButton: ExpendableFab(
           distance: 2.0,
+          icon: Icon(Icons.favorite), // Custom initial icon
+          //closeIcon: Icon(Icons.delete), // Custom close icon
```
Comment (ahmadalibaloch) [inline example/lib/main.dart]: It is included in docs, no need to have commented code

File: example/lib/main.dart [hunk 3]
```diff
@@ -56,7 +57,8 @@ class _MyAppState extends State<MyApp> {
   }
 
   toast(BuildContext context, String action) {
-    _scaffoldkey.currentState
-        ?.showSnackBar(SnackBar(content: Text("Selected $action")));
+    ScaffoldMessenger.of(context).showSnackBar(
```
Comment (ahmadalibaloch) [inline example/lib/main.dart:57]: 👍

**Comment by ahmadalibaloch** [comment]
Hi @HamadaHiro , thank you for the PR, left 2 minor comments and then good to go!

**Comment by csof3cen** [comment]
Ok, thanks for feedbacks. I applied changes.

**Review by ahmadalibaloch — DISMISSED** [review]

**Comment by csof3cen** [comment]
Hi, sorry. Commits was not signed. I done a rebase...

**Review by ahmadalibaloch — APPROVED** [review]

**Pull request merged**